components/user-claim-retriever/src/main/java/com/wso2telco/claims/RemoteClaimsRetriever.java

```diff
@@ -94,7 +97,7 @@ private String getclaimValue(Iterator<String> i, boolean isHashed, Map<String, O
         return requestedClaims;
     }
 
-    public static Map<String, Object> getTotalClaims(String operatorName, String msisdn) {
+    public static Map<String, Object> getTotalClaims(String operatorName, CustomerInfo customerInfo) {
```

Comment:
Is it necessary to make this a static method?

Reply:
fixed